### PR TITLE
Add order_rewards table

### PR DESCRIPTION
**Pull request by vkgnosis**
Part of https://github.com/cowprotocol/services/issues/606 .

In another PR we are going to be storing the rewards in this table.

### Test Plan

CI

### CODE REVIEW

**Review by nlordell — APPROVED** [review]
LGTM.

**Comment by nlordell** [comment]
cc @bh2smith for info on new table schema.

File: database/sql/V032__order_rewards.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Per order solver reward for https://forum.cow.fi/t/cip-draft-risk-adjusted-solver-rewards/1132 .
+-- Used in the weekly solver payout.
+CREATE TABLE order_rewards (
+    order_uid bytea NOT NULL,
+    auction_id bigint NOT NULL,
+    reward double precision NOT NULL,
```
Comment (bh2smith) [inline database/sql/V032__order_rewards.sql:6]: Any reason we don't use WEI here?

**Review by bh2smith — APPROVED** [review]
Looks great! Just a couple questions inline.

File: database/sql/V032__order_rewards.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    PRIMARY KEY(order_uid, auction_id)
```
[the remark this answers is not in the document]
Comment (vkgnosis) [reply database/sql/V032__order_rewards.sql:7]: The cip doesn't say how risk adjusted rewards is supposed to work with partially fillable orders so I'm not sure. One reasonable approach is that the reward given here is for the full order and discounted by how much of the order was filled, like for the protocol fee.

[the remark this answers is not in the document]
Comment (nlordell) [reply database/sql/V032__order_rewards.sql:7]: > I don't believe the contract enforces this.

No, but AFAIU, this is enforced with the HTTP solver `instance.json` and `result.json` format.

**Pull request merged**